src/visualization/types/Map/GeoOptions.tsx

```diff
@@ -1,4 +1,7 @@
 import React, {CSSProperties, FC} from 'react'
+import _ from 'lodash'
```

Comment:
please don't import all of lodash - just import the functions you need 🙏

Reply:
got it!